recipes/openorb-data/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "openorb-data" %}
+{% set version = "1.0.1.20181028" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
Should this URL now have `/oorb/` instead of `/mjuric/`?

Reply:
I'm building from my fork for now as I have a number of build system-related patches there that are destined for oorb/oorb but haven't been merged yet (https://github.com/oorb/oorb/pull/63). I thought about moving them all into patches (or one omnibus patch), but this seemed cleaner for now (the patch would be heavy). That said, happy to do it either way!

(again, I plan to switch to building from `oorb/oorb` as soon as the PR is merged).